src/test/rustdoc-gui/sidebar-mobile.goml

```diff
@@ -39,4 +39,4 @@ assert-position: ("#method\.must_use", {"y": 45})
 // Check that the bottom-most item on the sidebar menu can be scrolled fully into view.
 click: ".sidebar-menu-toggle"
 scroll-to: ".block.keyword li:nth-child(1)"
-assert-position: (".block.keyword li:nth-child(1)", {"y": 542.96875})
+assert-position: (".block.keyword li:nth-child(1)", {"y": 542.234375})
```

Comment:
With latest version, you can just check with `542` (which is much better for reading).

Reply:
(from https://github.com/rust-lang/rust/pull/93597)